frontend/cypress/support/index.ts

```diff
@@ -0,0 +1,8 @@
+declare namespace Cypress
+{
+	interface Chainable<Subject>
+	{
+		login() : Chainable<Subject>
+	}
+}
+Cypress.LocalStorage.clear();
```

Comment:
clearing the localstorage should be not done here - you could break features with that - introduce a logout command instead if you need this